release/models/system/openconfig-hashing.yang

```diff
@@ -0,0 +1,326 @@
+            data.";
+
+        container hashing-policies {
+            description
+              "Top level container for hashing, including configuration and
+              state data.";
+
+            list hashing-policy {
+                key "name";
+
+                description
+                  "The list of named policies to be used on the device.";
+
+                leaf name {
+                    type leafref {
+                        path "../config/name";
+                    }
+                    description
+                      "References the name of the hashing policy.";
+                }
+                container config {
+                    description
+                      "Configurable items at the global hash policy level.";
+
+                    uses hashing-policy-config;
+                }
+                container state {
+                    config false;
+                    description
+                      "Operational state data at the global hash policy
+                      level.";
+
+                    uses hashing-policy-config;
+                }
+                container hash-field-modes {
+                    description
+                        "Container for specifying inputs to be used when
+                        calculating the hash.";
+                    container config {
+                        description
+                            "Configurable items at the hashing inputs level.";
+                        uses hashing-inputs;
+                    }
+                    container state {
+                        config false;
+                        description "Operational state data at the hashing
+                            inputs level.";
+                        uses hashing-inputs;
+                    }
+                    container ipv4-fields {
+                        description "ipv4 hashing input fields.";
+                        uses ipv4-fields-top;
+                    }
+                    container ipv6-fields {
+                        description "ipv6 hashing input fields.";
+                        uses ipv6-fields-top;
+                    }
+                }
+            }
+        }
+    }
+
+    grouping supported-algorithms-state {
+        description
+            "Top-level container for the supported algorithms.";
+        leaf name {
+            type string;
+            description
+                "Name of the supported algorithm.";
+        }
+        leaf description {
+            type string;
+            description
+                "Description of the supported algorithm.";
+        }
+    }
+    grouping supported-algorithms-top {
+        description
+            "Top-level container of the supported algorithms for supported
+            algorithms.  The list of algorithms are expected to be defined by
+            the target device.";
+        container supported-algorithms {
```

Comment:
Can you rename the containers to include the word `hashing`.  So we get a path (combined with the augment suggestion below) which reads like:

`/system/hashing-algorithms/hashing-algorithm`